values.yaml

```diff
@@ -1,397 +1,165 @@
-  # 
-  # eg.
-  # nodeSelector:
-  #   disktype: ssd
-  nodeSelector: {}
-
-# certs -- Describes CAs that should be added to Coder services. These certs
-# are NOT added to environments.
-certs:
-  secret:
-    # certs.secret.name -- The name of the secret.
-    name: ""
-    # certs.secret.key -- The key in the secret pointing to the certificate
-    # bundle.
-    key: ""
-
-# namespaceWhitelist -- DEPRECATED: A list of additional namespaces to which
-# environments may be deployed. Use multiple workspace providers instead.
-# Existing environments within these namespaces will continue to function, but
-# new environments cannot be created within these namespaces.
-#
-# If you used this value before 1.17, you must keep it set to the same value
-# until all environments within these namespaces are deleted, then you can set
-# it back to [].
-namespaceWhitelist: []
-
-ssh:
-  # ssh.enable -- Enables accessing environments via SSH.
-  enable: true
-
-# serviceTolerations -- Tolerations are applied to all Coder managed services.
-# Each element is a toleration object. To set user environment tolerations see
-# environments.tolerations. See values.yaml for an example.
-#
-# e.g.
-# serviceTolerations:
-# - key: "my key"
-#   operator: "Exists"
-#   effect: "NoExecute"
-#   tolerationSeconds: 6000
-serviceTolerations: []
+# Contains fields related to the Postgres backend. If providing your own
+# instance, a minimum version of Postgres 11 is required with the contrib
+# package installed.
+postgres:
+  # postgres.host -- Host of the external PostgreSQL instance.
+  host: ""
+  # postgres.port -- Port of the external PostgreSQL instance.
+  port: ""
+  # postgres.user -- User of the external PostgreSQL instance.
+  user: ""
+  # postgres.database -- Name of the database that Coder will use.
+  # You must create this database first.
+  database: ""
+  # postgres.passwordSecret -- Name of an existing secret in the
+  # current namespace with the password of the PostgreSQL instance.
+  # The password must be contained in the secret field `password`.
+  # This should be set to an empty string if the database does not
+  # require a password to connect.
+  passwordSecret: ""
+  # postgres.sslMode -- Provides variable levels of protection for
+  # the PostgreSQL connection. For acceptable values, see:
+  #  https://www.postgresql.org/docs/9.1/libpq-ssl.html
+  sslMode: "require"
 
-# Services contains global configuration that applies to all Coder managed
-# services.
+  # postgres.default -- Configure a built-in PostgreSQL deployment.
+  default:
+    # postgres.default.enable -- Deploys a PostgreSQL instance. We recommend
+    # using an external PostgreSQL instance in production.
+    # If true, all other values are ignored.
+    enable: true
+    # postgres.default.image -- Injected by Coder during release.
+    image: ""
+    # postgres.default.storageClassName -- Set the storageClass to store
+    # the database.
+    storageClassName: ""
```

Comment:
Just want to clarify my understanding from the PR description above - since `StorageClassName` has been moved under `postgres`, that means I would need to set my PVC StorageClass at the workspace provider level, correct?

Reply:
Yup!